setup.py

```diff
@@ -13,7 +13,7 @@
 
 setup(
     name="plone.volto",
-    version="5.0.4.dev0",
+    version="5.0.4.dev1",
```

Comment:
You don't have to increment this; it is incremented to dev0 automatically after a release.
```suggestion
    version="5.0.4.dev0",
```